signPackages.builds

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
We should hopefully be able to remove this once we use the arcade signing.

Reply:
Agreed, I was just trying to get around using Arcade signing, so I needed a project with its own `build` target